CHANGELOG.md

```diff
@@ -16,6 +16,20 @@ should change the heading of the (upcoming) version to include a major version b
 
 -->
 
+# 5.6.1
+
+## @rjsf/core & @rjsf/utils
```

Comment:
You already mention `@rjsf/util` below
```suggestion
## @rjsf/core
```

Reply:
Ah, that's leftover from a previous set of CHANGELOG changes, my bad